### PR TITLE
Chassis intake

### PR DESCRIPTION
**Pull request by greenblitz4590**

### CODE REVIEW

**Review by tnrz135 — CHANGES_REQUESTED** [review]
A general comment:
Why the name ChassisIntake and not simply Intake? It doesn't use the chassis in the code (or something like that)

File: src/main/java/edu/greenblitz/robotName/subsystems/Intake/Intake.java [hunk 1]
```diff
@@ -0,0 +1,51 @@
...
+	private static IntakeInputsAutoLogged intakeInputs;
+	private static IIntake intake;
+	
+	private Intake() {
+		intake = IntakeFactory.create();
+		intakeInputs = new IntakeInputsAutoLogged();
+		intake.updateInputs(intakeInputs);
+	}
+	
+	public Intake getInstance() {
```
Comment (hillelv) [inline src/main/java/edu/greenblitz/robotName/subsystems/Intake/Intake.java:17]:
```suggestion
	public static Intake getInstance() {
		init();
		return instance;
	}

	public static void init(){
		if (instance == null) {
			instance = new Intake();
		}
	}
```

File: src/main/java/edu/greenblitz/robotName/subsystems/Intake/IntakeConstants.java [hunk 2]
```diff
@@ -0,0 +1,17 @@
+package edu.greenblitz.robotName.subsystems.Intake;
+
+import edu.wpi.first.math.controller.PIDController;
+
+public class IntakeConstants {
+	public static final PIDController MOTOR_PID_P = new PIDController(0, 0, 0);
```
Comment (hillelv) [inline src/main/java/edu/greenblitz/robotName/subsystems/Intake/IntakeConstants.java:6]: P?

File: src/main/java/edu/greenblitz/robotName/subsystems/Intake/RealIntake.java [hunk 3]
```diff
@@ -0,0 +1,29 @@
+package edu.greenblitz.robotName.subsystems.Intake;
+
+import com.revrobotics.CANSparkMax;
+import com.revrobotics.CANSparkMaxLowLevel;
+
+public class RealIntake implements IIntake {
```
Comment (hillelv) [inline src/main/java/edu/greenblitz/robotName/subsystems/Intake/RealIntake.java:6]: change to neo

**Review by hillelv — CHANGES_REQUESTED** [review]
Romy, i am natzier than raz

**Review by tnrz135 — APPROVED** [review]

**Pull request merged**